agent/containermetadata/write_metadata.go

```diff
@@ -0,0 +1,65 @@
+// Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Why is this file named 'write_metadata.go'? This does not really write metadata as such.